cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js

```diff
@@ -0,0 +1,319 @@
+        true,
+        DATASET.BREWERY_ADT,
+        RUN_TEMPLATE.BREWERY_PARAMETERS
+      ).then((value) => {
+        secondWorkspaceParentScenarioId = value.scenarioCreatedId;
+        ScenarioParameters.edit();
+        BreweryParameters.getStockInput().clear().type('400');
+        BreweryParameters.getRestockInput().clear().type('10');
+        BreweryParameters.getWaitersInput().clear().type('15');
+        Workspaces.getHomeButton().should('be.visible').click();
+        cy.go('back');
+        BreweryParameters.getStockInput().should('not.have.text', '400');
+        BreweryParameters.getRestockInput().should('not.have.text', '10');
+        BreweryParameters.getWaitersInput().should('not.have.text', '15');
+        ScenarioParameters.getParametersEditButton().should('not.be.disabled');
+        ScenarioParameters.edit();
+        BreweryParameters.getStockInput().clear().type('400');
+        BreweryParameters.getRestockInput().clear().type('10');
+        BreweryParameters.getWaitersInput().clear().type('15');
+        ScenarioParameters.updateAndLaunch();
+
+        Workspaces.getHomeButton().should('be.visible').click();
+
+        // check the first parent scenario is still running
+        Workspaces.selectWorkspace(REAL_BREWERY_WORKSPACE_ID);
+        ScenarioSelector.selectScenario(firstWorkspaceParentScenarioName, firstWorkspaceParentScenarioId);
+        Scenarios.getDashboardPlaceholder().should('have.text', SCENARIO_RUN_IN_PROGRESS);
+        ScenarioParameters.getLaunchButton().should('be.disabled');
+
+        // browse to the second parent scenario and check it still running with right values
+        route.browse(`${BREWERY_WORKSPACE_ID}/scenario/${secondWorkspaceParentScenarioId}`);
+        Scenarios.getDashboardPlaceholder().should('have.text', SCENARIO_RUN_IN_PROGRESS);
+        ScenarioParameters.getLaunchButton().should('be.disabled');
+        BreweryParameters.getStockInput().should('value', 400);
+        BreweryParameters.getRestockInput().should('value', 10);
+        BreweryParameters.getWaitersInput().should('value', 15);
+
+        Workspaces.getHomeButton().should('be.visible').click();
+        Workspaces.selectWorkspace(REAL_BREWERY_WORKSPACE_ID);
+
+        // create first child scenario, edit it without launching
+        Scenarios.createScenario(
+          firstWorkspaceChildScenarioName,
+          false,
+          firstWorkspaceParentScenarioName,
+          RUN_TEMPLATE.BASIC_TYPES
+        ).then((value) => {
+          firstWorkspaceChildScenarioId = value.scenarioCreatedId;
+          ScenarioParameters.edit();
+          BreweryParameters.switchToCustomersTab();
+          BreweryParameters.importCustomersTableData(CSV_VALID_WITH_EMPTY_FIELDS);
+
+          Workspaces.getHomeButton().should('be.visible').click();
+          Workspaces.selectWorkspace(BREWERY_WORKSPACE_ID);
+
+          // create second child scenario, edit it without launching
+          Scenarios.createScenario(
```

Comment:
It is really a good idea to include a scenario creation within another scenario creation promise ?

Reply:
I'm not sure but is there other way to get scenario id to use it through the test ?

Reply:
I think we can wrap a value in a promise to use it later but i'm no longer sure...

Reply:
The current implementation seems ok to me, IIRC we use the same nested `createScenario`s in another tests :slightly_smiling_face:  (there is also at least one test where we create scenarios before the `it` block of the test to store all scenarios ids)

Reply:
yes, in ScenarioLaunch, we create all tests in `before` but in this one, the idea is to perform different consecutive operation including create scenario during the test itself :slightly_smiling_face: